src/components/Typography/HX/HX.stories.tsx

```diff
@@ -0,0 +1,154 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { HX } from './HX'
+import { Icon } from '../../Icon'
+import { Typography } from '..'
+import { loremIpsum } from '../../../utils/loremIpsum'
+
+const customEllipsis = {
+  rows: 1,
+  suffix: ' END',
+  expandable: true,
+  symbol: 'Show more',
+}
+
+const customCopyable = {
```

Comment:
Can we move these resources to a Typography.mocks.ts file? in this way, we can also reuse them in tests.